### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
Prepping for plugin intake completion.  We need to update dependencies.

### CODE REVIEW

File: webapp/package-lock.json [hunk 1]
```diff
@@ -685,8 +685,7 @@
         },
         "lodash": {
           "version": "4.17.11",
-          "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.11.tgz",
```
Comment (levb) [inline webapp/package-lock.json]: curious, #iknownothing, seems like we are getting less strict?

**Comment by crspeller** [comment]
@jfrerich There look to be way more dependencies that need updating then this. What command did you run?

**Comment by jfrerich** [comment]
@crspeller, this is the result of running `npm audit fix` in the webapp/ directory.

**Comment by levb** [comment]
To be clear, there's no webapp yet, this is a "practice run", really.

**Comment by crspeller** [comment]
@jfrerich That really isn't a "dependency update" That command only updates for security issues. If you want to update dependencies you need to use something like `npm-check -E -u`

**Comment by crspeller** [comment]
@jfrerich Why is only the lock file updated? Did you forget to commit the `package.json`?

**Comment by jfrerich** [comment]
@crspeller I just added the package.lock file.

**Pull request merged**